UnitsNet.Tests/CustomCode/RotationalAccelerationTests.cs

```diff
@@ -30,5 +30,6 @@ public class RotationalAccelerationTests : RotationalAccelerationTestsBase
         protected override double DegreesPerSecondSquaredInOneRadianPerSecondSquared => 180 / Math.PI;
         protected override double RadiansPerSecondSquaredInOneRadianPerSecondSquared => 1;
         protected override double RevolutionsPerMinutePerSecondInOneRadianPerSecondSquared => 9.549296586;
+        protected override double RevolutionsPerSecondPerSecondInOneRadianPerSecondSquared => 1 / (2 * Math.PI);
```

Comment:
Use a constant here instead of calculation, preferably a constant found in some source. The idea is to double check we got the calculation right and easy to verify, as well as the conversion functions are in the right "direction" (not mixing up base to unit and unit to base, which is very easy to do).

Reply:
Used a constant value from [here](https://www.smartconversion.com/unit_conversion/angular_acceleration_converter.aspx) in the next commit